<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FournisseurWindow</class>
 <widget class="QMainWindow" name="FournisseurWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>914</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton
{
 background :  ; 
 border-radius : 60px ; 
}
QLabel
{
 color : #48494B  ; 
}
QLineEdit
{
 background:transparent ; 
border : none ;
color:#48494B ;
font-size : 20px ; 
border-bottom:0.5px solid #717072 ; 
}
QTabWidget::pane {
  top:-40px; 
} 
QTabBar::tab {
background-color: rgb(217, 221, 220);
border-radius : 15px;
border: 2px solid lightgray; 
border-color: rgb(255, 255, 255);
padding: 3px;
} 

QTabBar::tab:selected { 
background-color: rgb(217, 221, 220);
border: 3px solid lightgray; 
border-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 140, 0);
border-right-color: rgb(255, 140, 0);
margin-bottom: 0px; 
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1121</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 11pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(217, 221, 220);
border-radius : 35px ; </string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_CreateProvider">
     <attribute name="title">
      <string>Create Provider</string>
     </attribute>
     <widget class="QLabel" name="label_title">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Provider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_logo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineID">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>150</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Id Provider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>230</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_iconID">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>150</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/identifiant.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_type">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>250</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:  15px ; 
background-color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>Pharmacy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Restaurant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Clothes Shop</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_Name">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>340</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Name : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Place">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>430</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Place :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>390</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Provider Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_place">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>480</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Provider Place</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_save">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>460</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>460</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_iconName">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>390</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/la-personne.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_iconPlace">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>480</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/outil.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>90</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Pharmacy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>150</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Restaurant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>210</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Clothes shop</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_title_4">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>20</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 20pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Arduino :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_ProviderManagment">
     <attribute name="title">
      <string>Provider Managment</string>
     </attribute>
     <widget class="QLabel" name="label_titleListProviders">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>341</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>List Providers</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>130</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_iconSearch">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>130</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/search.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_update">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>560</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>560</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sort">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>240</y>
        <width>471</width>
        <height>231</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_place_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>420</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Provider Place</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Name_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>330</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Name : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Place_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>410</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Place :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>340</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Provider Name</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_type_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>260</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:  15px ; 
background-color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>Pharmacy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Restaurant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Clothes Shop</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_iconID_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/identifiant.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_iconPlace_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>420</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/outil.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_iconName_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>340</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/la-personne.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineID_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Id Provider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Id Provider : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="printButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_statistics">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Statistics</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_codeQr">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>470</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(217, 221, 220);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>530</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Code QR : </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Statistics">
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
     <widget class="QLabel" name="label_stat">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>961</width>
        <height>491</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>561</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
     <widget class="QLabel" name="label_title_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Statistics : </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="map">
     <attribute name="title">
      <string>Map</string>
     </attribute>
     <widget class="WebAxWidget" name="WebBrowser">
      <property name="control" stdset="0">
       <string>{8856F961-340A-11D0-A96B-00C04FD705A2}</string>
      </property>
      <property name="focusPolicy" stdset="0">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="geometry" stdset="0">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>991</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_title_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 28pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Map : </string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1212</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
  <customwidget>
   <class>WebAxWidget</class>
   <extends>QAxWidget</extends>
   <header>webaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
